### DISCUSSION TITLE
sugestion

### DISCUSSION THREAD

**Discussion by jboadas** [rbwhitaker/rbwhitaker.wikidot.com#7] [Ideas]
Hope this is the right place to write this.
As a suggestion, make the content of the wikidot cross platform, I mean add other procedures to use the tools on Linux , because Monogame also runs in Linux but the wiki only covers Visual Studio on Windows.

**Comment by rbwhitaker** [comment]
I missed this somehow. I definitely think this is a good idea. Most of the MonoGame code works fine in all places, but having a bit more about how to use another IDE like Visual Studio Code and getting all of the tools set up that way wouldn't be too big of a burden.
